tests/pass0/t04_indirect_call.test

```diff
@@ -0,0 +1,28 @@
+discard """
+  output: "(Done 100)"
+"""
+(TypeDefs
+  (Int 8)
+  (ProcTy (Type 0))
+)
+(GlobalDefs)
+(ProcDefs
+  (ProcDef (Type 1)
+    (Locals (Type 0))
```

Comment:
Did you mean to pass the `IntVal 100` as a parameter rather than have it hardcoded as it is now? Not sure what the `Locals` are for.

Reply:
Good catch. The locals are a leftover from a time where the VM validation layer erroneously required procedures to have at least one local, and I missed removing them before committing the tests.